jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Mainly a question, but do you see value in keeping mocha and jest side by side? Or should we consider migrating to jest? Or maybe just the parts that are using stubbing, replacing sinon so to speak?

Reply:
I'd do a gradual upgrade as the benefits of doing a full rewrite (less dev-dependencies) don't justify the cost (time consuming).